app/app/settings.py

```diff
@@ -155,6 +162,7 @@
 SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_KEY = env('OAUTH_AAD_KEY', None)
 SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_SECRET = env('OAUTH_AAD_SECRET', None)
 SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_TENANT_ID = env('OAUTH_AAD_TENANT', None)
+SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_WHITELISTED_DOMAINS = env('OAUTH_AAD_WHITELISTED_DOMAINS', '').split(',')
```

Comment:
Use can simplify this using [env.list](https://github.com/sloria/environs#supported-types).